backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Program.cs

```diff
@@ -55,4 +55,6 @@
     endpoinds.MapControllers();
 });
 
+app.UseCodiCoreContext();
```

Comment:
Can you send a link where you found this function?

Reply:
https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-easyspeak/blob/3059de461fa01b4ca6f6948b7ecb0058665e6100/backend/EasySpeak.Core/EasySpeak.Core.WebAPI/Extentions/ApplicationBuilderExtensions.cs#L10

Reply:
Ah, we already have it... Tanya...